[OSDOCS#17175]: Make changes as per the errors and warnings for CQA-2 (#114273)
openshift/openshift-docs: modules/running-compliance-scans.adoc

----
+
where:

> 

Reviewer: add a + here

----

> where:

Reviewer: You are getting these because you aren't correctly resolving the callout error.

----
+
where:

> `pvAccessModes.ReadWriteOnce`:: Specifies the Compliance Operator that creates a persistent volume (PV) that has the results of the scans. By default, the PV will use access mode `ReadWriteOnce` because the Compliance Operator cannot make any assumptions about the storage classes configured on the cluster. Additionally, `ReadWriteOnce` access mode is available on most clusters. If you need to fetch the scan results, you can do so by using a helper pod, which also binds the volume. Volumes that use the `ReadWriteOnce` access mode can be mounted by only one pod at time, so it is important to remember to delete the helper pods. Otherwise, the Compliance Operator will not be able to reuse the volume for subsequent scans.

Reviewer 2: ```suggestion
`Raw Result Storage.pvAccessModes.ReadWriteOnce`:: Specifies access mode for the PV created by the Compliance Operator with the results of the scans. By default, the PV will use access mode `ReadWriteOnce` because the Compliance Operator cannot make any assumptions about the storage classes configured on the cluster. Additionally, `ReadWriteOnce` access mode is available on most clusters. If you need to fetch the scan results, you can do so by using a helper pod, which also binds the volume. Volumes that use the `ReadWriteOnce` access mode can be mounted by only one pod at time, so it is important to remember to delete the helper pods. Otherwise, the Compliance Operator will not be able to reuse the volume for subsequent scans.
```

> `Raw Result Storage.Rotation`:: Specifies the Compliance Operator that keeps results of three subsequent scans in the volume; older scans are rotated.

Reviewer 2: ```suggestion
`Raw Result Storage.Rotation`:: Specifies that the Compliance Operator keeps the results of three subsequent scans in the volume; older scans are rotated.
```

> `Raw Result Storage.size`:: Specifies the Compliance Operator that allocates one GB of storage for the scan results.

Reviewer 2: ```suggestion
`Raw Result Storage.size`:: Specifies that the Compliance Operator will allocate one GB of storage for the scan results.
```

> `Raw Result Storage.storageClassName`:: Specifies the `scansetting.rawResultStorage.storageClassName` field specifies the `storageClassName` value to use when creating the `PersistentVolumeClaim` object to store the raw results. The default value is null, which will attempt to use the default storage class configured in the cluster. If there is no default class specified, then you must set a default class.

Reviewer 2: ```suggestion
`Raw Result Storage.Storage Class Name`:: Specifies the `storageClassName` value to use when creating the `PersistentVolumeClaim` object to store the raw results. The default value is null, which will attempt to use the default storage class configured in the cluster. If there is no default class specified, then you must set a default class.
```
